req2flatpak.py

```diff
@@ -771,6 +810,10 @@ def main():
     # generate flatpak-builder build module
     build_module = FlatpakGenerator.build_module(requirements, downloads)
 
+    if options.yaml:
+        yaml.dump(build_module, output_stream, default_flow_style=False)
+        parser.exit()
+
     # write output
     json.dump(build_module, output_stream, indent=4)
```

Comment:
Could you fix #45 by calling `build_module_as_str` instead of calling `json.dump` directly, thx